<launch>

 <!-- joystick -> vectorCommand-->
  <node pkg="rov_control_interface" type="drive_control" name="drive_control"/>

 <!-- Joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="rov_joy" >
    <remap from="joy" to="joy/joy1"/>
    <param name="dev" type="string" value="/dev/input/by-id/usb-Logitech_Logitech_Extreme_3D-joystick" />
    <param name="deadzone" value="0.1" />
    <param name="autorepeat_rate" value="2"/>
  </node>

  <!--  Throttle joy node -->
    <node respawn="true" pkg="joy"
          type="joy_node" name="rov_throttle" >
      <remap from="joy" to="joy/joy2"/>
      <param name="dev" type="string" value="/dev/input/by-id/usb-Thrustmaster_TWCS_Throttle-joystick" />
      <param name="deadzone" value="0.1" />
      <param name="autorepeat_rate" value="2"/>
    </node>

</launch>
